### PR TITLE
Travis 2.6

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

**Review by dnsl48 — CHANGES_REQUESTED** [review]
https://github.com/silverstripe/cwp-core/pull/88#discussion_r443905185

File: .travis.yml [hunk 1]
```diff
@@ -8,13 +8,13 @@ services:
 matrix:
   include:
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
```
Comment (dnsl48) [inline .travis.yml]: It used to be `4.6.x-dev`, and not `4.x-dev`

File: .travis.yml [hunk 2]
```diff
@@ -24,7 +24,11 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-core:4.6.x-dev silverstripe/versioned:1.6.x-dev
+  - composer require --no-update silverstripe/recipe-core:$RECIPE_VERSION
+  # silverstripe/versioned must be 1.x-dev (or 1.n.x-dev), not ^1
```
Comment (dnsl48) [inline .travis.yml]: Would be nice to clarify why exactly it must be `.x-dev` and not `^1.`
Are there some technical restrictions for that?
`"minimum-stability": "dev"` seems to be assigned in `composer.json`

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -24,7 +24,11 @@ before_script:
...
+  - composer require --no-update silverstripe/versioned:1.x-dev
```
Comment (dnsl48) [inline .travis.yml]: It used to be `1.6.x-dev`, not `1.x-dev`

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (emteknetnz) [reply .travis.yml]: Updated to `1.6.x-dev`

File: .travis.yml (hunk 2 above)
Comment (emteknetnz) [reply .travis.yml]: Have added $VERSIONED_VERSION as a global variable, should be a little less confusing now what's going on

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml]: Updated to `4.6.x-dev`

**Pull request closed without merging**